app/dashboard/agents/invoice/page.tsx

```
@@ -1,7 +1,398 @@
+
+    try {
+      const response = await fetch("/api/agents/invoice/postInvoice", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ input }),
+      });
+
+      // Parse the response
+      let data;
+      const contentType = response.headers.get("content-type");
+      if (contentType && contentType.includes("application/json")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)